src/list.json

```diff
@@ -203,6 +203,17 @@
         "description": "Meilisearch is once again back for Hacktoberfest! We look back on previous editions while getting ready for this year’s, armed with brand new common goals and plenty of exclusive new swag for our contributors.",
         "no_of_prs": 1,
         "link": "https://blog.meilisearch.com/contribute-hacktoberfest-2022/"
+      },
+      {
+        "organization": "Metafy",
+        "org_url": "https://github.com/metafy-social/.github/blob/main/profile/README.md",
+        "tags": [
+          "NFT",
+          "T-shirt"
```

Comment:
```suggestion
          "T-shirt" => "Tshirt"
```

Reply:
Later on Please Re-style it using this command
`git checkout <your branch>`
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2340980/patch | git am`
`git push `

Reply:
Ok cool